js/ui/html_editor/modules/toolbar.js

```diff
@@ -472,11 +473,28 @@ if(Quill) {
             return $container;
         }
 
+        _checkItemOptions(item) {
+            const optionsInfo = [{
+                newName: 'name',
+                oldName: 'formatName'
+            }, {
+                newName: 'acceptedValues',
+                oldName: 'formatValues'
+            }];
+
+            each(optionsInfo, (index, optionName) => {
+                if(isObject(item) && isDefined(item[optionName.oldName])) {
```

Comment:
`isObject(null)` gives true. Seems like `Cannot read property of null` can be thrown here `optionName.oldName`

Reply:
our isObject returns false in this case 
console.log(isObject(null));  // false